pdf_build_src/header_setup.tex

```diff
@@ -25,3 +25,5 @@
     linewidth=\textwidth,
     basewidth=0.5em
 }
+
+\usepackage[a4paper, margin=1.5in]{geometry}
```

Comment:
not sure how, but this change may break several tables :man_shrugging: 

- see p.63 and down in [previous pdf](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/5bac04e913babe000192f2a5/5f5a9975636a043aa45b3639-0-build/artifacts/bids-spec.pdf?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20200915T075534Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20200915%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=094061e0fc6a01337b75f60bf4a8229fa863e3f9a30ba1455be48a8a1cbc5e32)
- compared to p49 and scroll down in [this pdf](https://circle-production-customer-artifacts.s3.amazonaws.com/picard/forks/5bac04e913babe000192f2a5/151142273/5f60717f651d6e58c50eae60-0-build/artifacts/bids-spec.pdf?X-Amz-Algorithm=AWS4-HMAC-SHA256&X-Amz-Date=20200915T075059Z&X-Amz-SignedHeaders=host&X-Amz-Expires=60&X-Amz-Credential=AKIAJR3Q6CR467H7Z55A%2F20200915%2Fus-east-1%2Fs3%2Faws4_request&X-Amz-Signature=ca59a73cce238fa9f2e220d356e9a4a82ef22a974f62aebc94606da29206e2b4)

@sebastientourbier do you have a clue about what's going on here?

I'd like the tables to be nice again, and potentially also decrease the marging of the whole document by a bit

Reply:
fixed it, was a tricky mix of bugs, described it in the OP.